src/Compiler/SyntaxTree/SyntaxTreeOps.fs

```diff
@@ -1005,6 +1005,16 @@ let parsedHashDirectiveArguments (input: ParsedHashDirectiveArgument list) (lang
             | false -> None)
         input
 
+let parsedHashDirectiveArgumentsNoCheck (input: ParsedHashDirectiveArgument list) (langVersion: LanguageVersion) =
+    List.choose
+        (function
```

Comment:
List.choose with all branches returning "Some" and never returning "None" -> that's just List.map

Reply:
Thanks. Done.